### PR TITLE
update lesson to explain owning side of many-to-many

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,27 +4,39 @@
...
-entity `STUDENT_DATA_SUBJECT` to keep track of the relationships. Note the
-entity has a composite primary key `(student, subject)`. The updated ER model
-that contains the new entity is shown below:
+![Updated many to many relationship join table](https://curriculum-content.s3.amazonaws.com/6036/java-mod-5-jpa/student_subject_join_table.png)
 
-![Updated many to many relationship join table](https://curriculum-content.s3.amazonaws.com/6036/java-mod-5-jpa-manytomany/student_subject_withjointable.png)
+However, the join table representing the `Student_Subject` entity is automatically
+created by JPA, so we only need to create the `Subject` entity class in Java and then
+define the many-to-many relationship between `Student` and `Subject` using the `@ManyToMany`
+annotation based on model defined in the original ERD:
```
Comment (kcleland0818) [inline README.md]: I think we need the word "the" between "on" and "model"; "using the `@ManyToMany` annotation based on model defined in the original ERD".

File: README.md [hunk 2]
```diff
@@ -84,7 +85,179 @@ The project directory structure should look like this:
...
+import javax.persistence.Persistence;
+import java.time.LocalDate;
+
+public class JpaCreate {
+    public static void main(String[] args) {
+        // create a new student instance
+        Student student1 = new Student();
+        student1.setName("Jack");
+        student1.setDob(LocalDate.of(2000,1,1));
+        student1.setStudentGroup(StudentGroup.ROSE);
```
Comment (kcleland0818) [inline README.md:107]: Getting Titanic vibes with "Jack" being in a group called "Rose" lol

**Review by kcleland0818 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: yes, thanks!

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md:107]: ha!

**Review by kcleland0818 — APPROVED** [review]

**Comment by developerjaym** [comment]
Consider adding the ds_store file to the gitignores

File: README.md [hunk 3]
```diff
@@ -185,7 +366,7 @@ We could alternatively have established the relationship by adding a student to
 subject's list of students:
 
 ```java
-// create students >-< subjects association
+// create students <> subjects association
```
Comment (developerjaym) [inline README.md:369]: I can't comment on the exact line so it's not actually part of the diff, but a few lines down, it says
```We only need to add to one list to add the association in the database.```

But it looks like the code adds to both lists (addStudent method calls the addSubject method).

File: README.md [hunk 4]
```diff
@@ -584,26 +768,41 @@ public class JpaRead {
 
 We have learned how to create many-to-many relationships in this lesson. 
 
-A student is associated with many subjects, thus the `Student` class stores
-the relationship as:
+- Pick one entity to be the owner of the relationship.
```
Comment (developerjaym) [inline README.md:769]: Great conclusion!

**Review by developerjaym — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (linda-seiter) [reply README.md:369]: Removed the comment about adding to one list.

File: README.md (hunk 4 above)
Comment (linda-seiter) [reply README.md:769]: thanks!

**Review by developerjaym — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**